Metadata/MetadataConverter.php

```diff
@@ -80,11 +80,15 @@ public function convert(ClassHierarchyMetadata $metadata)
             $definition->setProperties($classMetadata->properties);
 
             if (null !== $classMetadata->decorates) {
+                if (!isset($classMetadata->decorationInnerName) && isset($classMetadata->decoration_inner_name)) {
```

Comment:
isset already checks that and has the advantage to keep working if the property is removed later...
I'm really checking "is this property used?". I shouldn't crash if the property doesn't exist...